packages/xo-server-transport-nagios/src/index.js

```diff
@@ -2,11 +2,23 @@ import crc32 from 'buffer-crc32'
 import net from 'net'
 import { Buffer } from 'buffer'
 import { createLogger } from '@xen-orchestra/log'
+// eslint-disable-next-line n/no-extraneous-import
+import { compileTemplate } from '@xen-orchestra/template'
 
 const { debug, warn } = createLogger('xo:server:transport:nagios')
 
 // ===================================================================
 
+const hostDescription = `Host name on Nagios.
+  Leave empty if the host name equals the vm name (the default configuration).
+  Otherwise, you could choose a custom name but the template '{vm.name_label}' must
+  be included. For example: 'xo-backup-{vm.name_label}'`
```

Comment:
- Remove unwanted indent in the markdown text
- New lines are ignored, if you want to separate text in blocks, use blank lines to force paragraphs
- Use backticks for code format

```suggestion
const hostDescription = `Host name on Nagios.

Leave empty if the host name equals the vm name (the default configuration).

Otherwise, you could choose a custom name but the template \`{vm.name_label}\` must  be included. For example: \`xo-backup-{vm.name_label}\``
```